src/app/pages/legacy-trip/legacy-trip.page.ts

```diff
@@ -174,6 +175,9 @@ export class LegacyTripPage implements OnInit, OnDestroy {
               (error) => {
                 this.loggingService.error(error, 'Error when starting trip', DEBUG_TAG);
                 this.tripLoggerService.showTripErrorMessage(true);
+                this.ngZone.run(() => {
+                  this.isLoading = false;
```

Comment:
I stedet for å legge til mer ngzone-greier, kanskje bruk et/en signal?
Vi kan egentlig deaktivere change detection med ngzone nå, hvis vi vil.